ext/drbbdup/drbbdup.h

```diff
@@ -54,6 +54,7 @@ extern "C" {
 typedef enum {
     DRBBDUP_SUCCESS,                       /**< Operation succeeded. */
     DRBBDUP_ERROR_INVALID_PARAMETER,       /**< Operation failed: invalid parameter. */
+    DRBBDUP_ERROR_UNSET_FEATURE,           /**< Operation failed: feature not set. */
```

Comment:
Better to append to the end to not break compat.

Reply:
Done.